run_at.py

```diff
@@ -0,0 +1,16 @@
+import sys
+import os
+
+# Alias for ../TestRunners/AnnotationTestRunner/run_test.py
```

Comment:
Why is this necessary?

Reply:
Purely convenience. Same reason for `run_pt.py`; easier to remember and type out `python ../run_at.py testname` than `python ../TestRunners/AnnotationTestRunner/run_test.py testname`, though I can always `.gitignore` them if you feel this does not justify their existence

Reply:
That's reasonable. Please add this logic to the comment at the start of this file.